assets/templates/main.tmpl

```diff
@@ -29,6 +29,10 @@
       {{- end }}
     {{ end }}
 
+    {{if eq .Metadata.Title "Feedback"}}
+      <link rel="canonical" href="ons.gov.uk/feedback">
```

Comment:
Actually, it might be worth using the `.SiteDomain` variable that we use in the config, instead of hardcoding "ons.gov.uk" - give me a shout if you want to go through this